### ISSUE TITLE
Portal Search - KEGG

### ISSUE THREAD

**Issue by ssarrafan** [microbiomedata/nmdc-server#362]
Enable search by modules and the 3 Pathway Levels. KO term is very granular. Oftentimes we're interested in understanding whether there's presence of an entire pathway (sometimes this may be specific, like "fatty acid metabolism" which is a level 3 pathway, but other times we may be interested in "metabolism" in general which is a level 1 pathway)

Priority - High
Urgency - Medium

**Comment by kfagnan** [comment]
In today's meeting Jeff indicated Jon has the information needed to work on this issue. So we'll leave in this sprint for now.

**Comment by ssarrafan** [comment]
This issue is dependent on the fix to be done on another issue - https://github.com/microbiomedata/nmdc-metadata/issues/342 per @pvangay

**Comment by ssarrafan** [comment]
@zachmullen @subdavis I'm just checking to see if you have everything you need for this issue.  Would you be able to get this completed and available in the portal by the end of this September?  Please let me know if you need anything from anyone on the team or if you're blocked by anything.

**Comment by zachmullen** [comment]
Hi @ssarrafan , I am not aware of any blockers at this time but I'm in a preliminary stage on this. I will raise any issues if I encounter them.

**Comment by ssarrafan** [comment]
Moving to October's sprint since this is being tested now and part of the October release.

**Comment by ssarrafan** [comment]
Released yesterday to production. Closing this issue.
